lib/core/src/lib/common/services/thumbnail.service.ts

```diff
@@ -158,7 +158,11 @@ export class ThumbnailService {
         'save-as': './assets/images/save-as.svg',
         save: './assets/images/save.svg',
         task: './assets/images/task.svg',
-        'multipart/related': './assets/images/ft_ic_website.svg'
+        'multipart/related': './assets/images/ft_ic_website.svg',
+        side_nav_collapse: './assets/images/icon-expand-navigation.svg',
```

Comment:
Please change `_` to `-` to comply with already created icons

Reply:
changed